manifests/pipecd/values.yaml

```diff
@@ -107,3 +107,12 @@ cors:
     - "http://localhost:9090"
 
 quickstart: false
+
+monitoring:
+  enabled: false
+
+# All directives inside this section will be directly sent to the prometheus chart.
+# Head to the below link to see all available values.
+# https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus/values.yaml
+# TODO: Prepare the default prometheus.yml
+prometheus: {}
```

Comment:
How about moving this into the "monitoring" group?

Reply:
It is. Initialy I wanted to do so but couldn't. Tell me how to do it if you already know! If none I will give it a try to look into them.

Reply:
I little poked around how to do it, but none of them worked as well as we'd like.

Reply:
👍  It seems there is no way for that.